app.js

```diff
@@ -16,25 +16,25 @@ var app = express();
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'jade');
 
-// uncomment after placing your favicon in /public
-//app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
+
+
 app.use(logger('dev'));
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.urlencoded({extended: false}));
 app.use(cookieParser());
 app.use(session({
   secret: 'secret',
   name: 'browser-calls',
-  cookie: { maxAge: 60000 },
+  cookie: {maxAge: 60000},
   resave: true,
-  saveUninitialized: true
+  saveUninitialized: true,
```

Comment:
I believe this is to avoid having to replace the line in git if you ever have to add another item to the object just because a comma was added to the end of the line.